server/src/main/java/io/seata/server/coordinator/DefaultCore.java

```diff
@@ -82,6 +82,7 @@ public Long branchRegister(BranchType branchType, String resourceId, String clie
                 globalSession.addBranch(branchSession);
             } catch (RuntimeException ex) {
                 branchSession.unlock();
+                LOGGER.error("Failed to add branchSession to globalSession : " + ex);
```

Comment:
+ex ？

Reply:
Just add error log,  easy to find problems,The real cause is still being investigated